.github/workflows/dev-green.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy to Dev-green
+
+on:
+  workflow_dispatch:
+  push:
+    branches: master
```

Comment:
This will automatically deploy on merges to master, right? Is that what we want for this env?

Reply:
You're correct. Actually I don't think we want that and should only deploy to dev-green when manually triggered. Will remove the `push:` option